RPS src/RPS.js

```diff
@@ -0,0 +1,19 @@
+export default function compPlay() {
+    let compRandom = Math.round(Math.random() * 2);
+    const compInfo = compMove(compRandom);
+    return compInfo;
+}
+
+
+export function compMove(compRandom) {
+    if(compRandom === 0) {
+        return 'stone_PNG13612';
```

Comment:
you should just use 'stone' or 'rock' as the strings here for readability, and rename the image files themselves.